stages/org.osbuild.coreos.live-iso

```diff
@@ -0,0 +1,763 @@
+#!/usr/bin/python3
```

Comment:
Note to self: I should add a comment here that provides the history of this file so future git archeologists know to look at the coreos-assembler repo.